packages/lit-dev-content/site/docs/tools/server-side-rendering.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Server-side Rendering
+eleventyNavigation:
+  key: Server-side rendering
+  parent: Tools
+  order: 9
+---
+
+Server-side rendering (SSR) Lit templates and elements is possible with the [`@lit-labs/ssr`](/docs/libraries/labs/) package. As with all `@lit-labs` packages, it is considered experimental and not ready for production use but we encourage you to try it out and file any issues for bugs or feature requests.
```

Comment:
"SSR... is possible" is a bit vague. Let's get more specific here, like:

> Lit SSR lets you render Lit components and templates to static HTML markup in non-browser JavaScript environments like Node.

Then I'd include some brief mentions of:
* Some qualities of Lit SSR
* Why you might use SSR
* That there are restrictions and lots of consideration
* That you probably want some integration with a server or other tool

Some potential phrasing:

Lit SSR works without emulating the browser's DOM, and takes advantage of Lit's declarative template format to support streaming, asynchronous rendering.

SSR can help make pages work without JavaScript, either for robustness or performance reasons, or to work with web crawlers and bots that don't run a full browser. Some sites can render faster if they render static markup first, then load client-side later, and optionally hydrate components.

Rendering web components in a server environment incurs some restrictions on component code because most browser DOM APIs are not available, and some concepts like certain lifecycle callbacks don't apply to a server, and streaming rendering occurs in a strict top-down ordering.

Read on for more... (or something like that)

Reply:
> Lit SSR works without emulating the browser's DOM, and takes advantage of Lit's declarative template format to support streaming, asynchronous rendering.

We don't actually support asynchronous rendering in SSR yet though, right?

> SSR can help make pages work without JavaScript

Currently only for Chromium though since declarative shadow DOM polyfill requires JavaScript. 😞 Do we need to mention that?

Reply:
We definitely should mention that we rely on declarative shadow root, both in the positive sense that it enables shadow DOM w/o JS, and to mention that it requires a polyfill on Safari and FF.